### PR TITLE
Fix gitops deploy

### PR DESCRIPTION
**Pull request by emosbaugh**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -274,15 +274,15 @@ jobs:
             if [ "${CIRCLE_PROJECT_USERNAME}" == "replicatedhq" ]; then
               sed -i "s/DYNAMIC_IMAGE_TAG/${CIRCLE_SHA1:0:7}/g" ~/project/kustomize/overlays/production/kustomization.yaml
               echo 'github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==' >> ~/.ssh/known_hosts
-              cd ~ && git clone git@github.com:replicatedcom/gitops-deploy
+              cd ~ && git clone --single-branch -b release git@github.com:replicatedcom/gitops-deploy
```
Comment (dexhorthy) [inline .circleci/config.yml]: is there a reason not to `--single-branch -b master` on the staging deploy?

Comment (emosbaugh) [reply .circleci/config.yml]: @dexhorthy i copied this from vendor api
https://github.com/replicatedcom/vendor-api/blob/master/.circleci/config.yml#L122

File: .circleci/config.yml [hunk 2]
```diff
@@ -179,56 +169,30 @@ jobs:
...
-              mkdir -p ~/gitops-deploy/replicated-installer
-
-              ~/kustomize build ~/project/kustomize/overlays/staging > ./gitops-deploy/replicated-installer/replicated-installer.yaml
-              cd ~/gitops-deploy && \
-              git config user.email "circleci@replicated.com" && \
-              git config user.name "CircleCI Build" && \
-              git add . && \
-              git commit -m ${CIRCLE_COMPARE_URL} && [ $? -eq 0 ] && git push origin master
-            fi
+            pushd kustomize/overlays/staging
```
Comment (dexhorthy) [inline .circleci/config.yml]: I don't think you need this

File: .circleci/config.yml [hunk 3]
```diff
@@ -245,67 +209,51 @@ jobs:
...
-              mkdir -p ~/gitops-deploy/replicated-installer
-
-              ~/kustomize build ~/project/kustomize/overlays/production > ./gitops-deploy/replicated-installer/replicated-installer.yaml
-              cd ~/gitops-deploy && \
-              git config user.email "circleci@replicated.com" && \
-              git config user.name "CircleCI Build" && \
-              git add . && \
-              git commit -m ${CIRCLE_COMPARE_URL} && [ $? -eq 0 ] && git push origin release
-            fi
+            pushd kustomize/overlays/production
```
Comment (dexhorthy) [inline .circleci/config.yml]: Ditto

**Pull request merged**